acp/ideas_module.php

```diff
@@ -313,6 +322,9 @@ public function select_ideas_topics_poster($value, $key)
 	 */
 	public function set_ideas_forum_permissions($value, $key)
 	{
-		return '<input class="button2" type="submit" id="' . $key . '" name="' . $key . '" value="' . $this->language->lang('RUN') . '" />';
+		return '
+			<form id="acp_phpbb_ideas_forum_setup" method="post" action="' . $this->u_action . '" data-ajax="true">
```

Comment:
Nope as the form tag is being closed here https://github.com/phpbb/phpbb-ideas/pull/39/files#diff-b7e8b82834b406f772db3f62d5ceba57R38